### PR TITLE
Plancard style2

### LINKED ISSUE #111: Plan Card Design Updates
Adding a top nav bar to hold the Title and Edit icon.
Adding a placeholder description field to the bottom section below the thumbnail.
Increasing text size and ensuring long titles use the overflow property and return an ellipsis.

### PR DESCRIPTION
**Pull request by RDiMag**
Closes #111

### CODE REVIEW

**Comment by RDiMag** [comment]
I'm not sure how to fix the error with Netlify.
"Line 20:3:  'isAdmin' PropType is defined but prop is never used  react/no-unused-prop-types"

I tried removing 'isAdmin' completely from this file, but that breaks some of the Editor files.

File: src/components/PlanCard/PlanCard.styled.tsx [hunk 1]
```diff
@@ -40,16 +49,39 @@ export const PlanNav = styled.section`
   width: 100%;
   height: 25%;
   background: ${({ theme }) => theme.color.white};
+  color: ${({ theme }) => theme.color.primary};
+  font-size: 0.5em;
```
[the remark this answers is not in the document]
Comment (RDiMag) [reply src/components/PlanCard/PlanCard.styled.tsx:53]: Thanks!

**Review by Marcellia — APPROVED** [review]

File: src/components/PlanCard/PlanCard.styled.tsx [hunk 2]
```diff
@@ -1,29 +1,38 @@
...
   padding: 15px;
+  align-items: center;
 `;
 export const PlanId = styled.p`
-  color: ${({ theme }) => theme.color.white};
-  font-size: 1.1rem;
-  font-weight: 700;
+  font-size: 1.3rem;
+  font-weight: 600;
+  padding-left: 6px;
```
Comment (theodorusandi) [inline src/components/PlanCard/PlanCard.styled.tsx:32]: I think we should do em / rem for padding for consistency

Comment (RDiMag) [reply src/components/PlanCard/PlanCard.styled.tsx]: Done! Thank you.

**Pull request merged**